src/content/manifest/okteto-variables.mdx

```diff
@@ -1,15 +1,36 @@
 ---
 title: Okteto Variables
-description: Okteto Variables allows you to save application configuration in Okteto, and automatically inject them during deployment time
-sidebar_label: Okteto variables
+description: Okteto Variables allows you to save application configuration in Okteto and automatically inject them during deployment time
+sidebar_label: Variables
 id: okteto-variables
 ---
 
 import Image from "@theme/Image";
 
 Application configuration should be passed at deployment time, not hardcoded in your code.
 
-Okteto Variables allows you to save application configuration in Okteto, and automatically inject it during deployment time.
+Okteto variables help you save application configuration in Okteto and automatically inject it during deployment time.
+
+There are a few different ways to leverage variables. In short:
+
+- You can define variables manually in the manifest or in the Okteto UI
+- There are some default variables built in to the Okteto platform
+- All variables used in the Okteto manifest are automatically expanded
+- Variables referenced in the `deploy` and `destroy` sections are automatically injected into those environments
```

Comment:
@pchico83 is this an accurate representation? It felt right to me but wondering if I've captured your suggestion here

Reply:
Maybe something like `All variables are injected as runtime environment variables in the execution of your "deploy" and "destroy" scripts`?

Reply:
Done (locally) 👍 Will push in next commit